app/views/kaminari/_page.html.erb

```diff
@@ -0,0 +1,16 @@
+<%# Link showing page number
+  - available local variables
+    page:          a page object for "this" page
+    url:           url to this page
+    current_page:  a page object for the currently displayed page
+    total_pages:   total number of pages
+    per_page:      number of items to fetch per page
+    remote:        data-remote
+-%>
+<span class="page<%= ' current' if page.current? %>">
+  <% if page.current? %>
+    <span><%= page %></span>
+  <% else %>
+    <%= govuk_link_to page, url, remote: remote, rel: page.rel %>
```

Comment:
Why do we have remote here? I don't think we are using ajax to load the link

Reply:
This was generated, but I don't think we are using ajax https://github.com/kaminari/kaminari#ajax-links-crazy-simple-but-works-perfectly